### PR TITLE
Add SQLServer Revoke statement

### PR DESCRIPTION
**Pull request by ThanoshanMV**
For #6478 

Changes proposed in this pull request:
- I've proofread the SQLServer [REVOKE](https://docs.microsoft.com/en-us/sql/t-sql/statements/revoke-transact-sql?view=sql-server-ver15) statement and added test cases.
- I've added `grantClassPrivileges`, `grantOnClassClause`, `grantClassTypePrivileges`, and `grantOnClassTypeClause` rules in the `revoke` rule as they are same by definition. After proofreading `deny`, I'll rename them to generic names.

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/sqlserver/dcl/SQLServerRevokeStatement.java [hunk 1]
```diff
@@ -17,13 +17,21 @@
...
+import java.util.List;
+
 /**
  * SQLServer revoke statement.
  */
 @ToString
+@Getter
 public final class SQLServerRevokeStatement extends RevokeStatement implements SQLServerStatement {
+    
+    private final List<ColumnSegment> columns = new LinkedList<>();
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/sqlserver/dcl/SQLServerRevokeStatement.java:36]: @ThanoshanMV Can we use Collection<ColumnSegment> here?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/sqlserver/dcl/SQLServerRevokeStatement.java (hunk 1 above)
Comment (ThanoshanMV) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/sqlserver/dcl/SQLServerRevokeStatement.java]: Hi @strongduanmu, I've updated it.

Comment (strongduanmu) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/sqlserver/dcl/SQLServerRevokeStatement.java]: > Hi @strongduanmu, I've updated it.

Thank you @ThanoshanMV, but I found travis ci build error, can you check it on your machine?

**Review by strongduanmu — APPROVED** [review]
@ThanoshanMV Thank you, this pr is perfect!

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/sqlserver/dcl/SQLServerRevokeStatement.java (hunk 1 above)
Comment (ThanoshanMV) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/sqlserver/dcl/SQLServerRevokeStatement.java]: @strongduanmu I tried to search and have come up with a possible reason! Please let me know if this is the real reason behind ;) We're changing a list of `ColumnSegment` from `List<ColumnSegment>` to `Collection<ColumnSegment>` in the `SQLServerRevokeStatement` as `((CollectionValue<ColumnSegment>) visit(each)).getValue()`'s compile-time type (at line 141 in `SQLServerDCLStatementSQLVisitor.java`) will be `Collection<ColumnSegment>`. The `getColumns()`'s compile-time type will be `Collection<ColumnSegment>`.

So basically, we're getting a `Collection<ColumnSegment>` and adding a `Collection<ColumnSegment>` into it at compile-time. To keep consistency among the datatype at compile-time, we're changing like that. If this is not, please let me know the actual reason :)
